src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="../src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews-number">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__prices">
+        <span class="card__prices-text">
+          Price:
+        </span>
+        <span class="card__prices-numbers">
+          $2,199
+        </span>
+      </div>
+      <a href="#"
+      class="card__button"
+      data-qa="hover">
+        Buy
+      </a>
```

Comment:
```suggestion
      <a
        href="#"
        class="card__button"
        data-qa="hover"
      >
        Buy
      </a>
```
you should always fix all similar cases (in the previous review there were a few comments about formatting)